generators/client/templates/src/main/webapp/swagger-ui/_index.html

```diff
@@ -85,18 +85,18 @@
                     window.swaggerUi.api.clientAuthorizations.add("key", apiKeyAuth);
                 <% } %>
                 <% if (authenticationType == 'oauth2') { %>
-                    var authToken = JSON.parse(localStorage.getItem("ls.token")).access_token;
+                    var authToken = JSON.parse(localStorage.getItem("jhi-authenticationToken")).access_token;
                     var apiKeyAuth = new SwaggerClient.ApiKeyAuthorization("Authorization",
                             "Bearer " + authToken, "header");
                     window.swaggerUi.api.clientAuthorizations.add("key", apiKeyAuth);
                 <% } %>
                 <% if (authenticationType == 'xauth') { %>
-                    var authToken = JSON.parse(localStorage.getItem("ls.token")).token;
+                    var authToken = JSON.parse(localStorage.getItem("jhi-authenticationToken")).token;
                     var apiKeyAuth = new SwaggerClient.ApiKeyAuthorization("X-Auth-Token", authToken, "header");
                     window.swaggerUi.api.clientAuthorizations.add("key", apiKeyAuth);
                 <% } %>
                 <% if (authenticationType == 'jwt') { %>
-                    var authToken = JSON.parse(localStorage.getItem("ls.authentication-token"));
+                    var authToken = JSON.parse(localStorage.getItem("jhi-authenticationToken"));
```

Comment:
Bug if user doesn't check "Remember me" at login (token stored in sessionStorage).